### PR TITLE
Update from version 14 to version 20

### PR DESCRIPTION
**Pull request by tpmccallum**
Signed-off-by: tpmccallum tim.mccallum@fermyon.com
Closes https://github.com/fermyon/developer/issues/1318
Content must go through a pre-merge checklist.

## Pre-Merge Content Checklist

This documentation has been checked to ensure that:

- [x] The `title`, `template`, and `date` are all set
- [x] Does this PR have a new menu item (anywhere in `templates/*.hbs` files) that points to a document `.md` that is set to publish in the future? If so please only publish the `.md` and `.hbs` changes in real-time (otherwise there will be a menu item pointing to a `.md` file that does not exist)
- [x] File does not use CRLF, but uses plain LF (hint: use `cat -ve <filename> | grep $'\r' | wc -l` and expect 0 as a result)
- [x] Has passed [`bart check`](https://developer.fermyon.com/bartholomew/quickstart)

✅ content/wasm-languages/c-lang.md

- [x] Has been manually tested by running in Spin/Bartholomew (hint: use `PREVIEW_MODE=1` and run `npm run styles` to update styling)
- [x] Headings are using Title Case
- [x] Code blocks have the programming language set to properly highlight syntax and the proper copy directive
- [x] Have tested with `npm run test` and resolved all errors
- [x] Relates to an existing (potentially outdated) blog article? If so please add URL in blog to point to this content.

### CODE REVIEW

File: content/wasm-languages/c-lang.md [hunk 1]
```diff
@@ -72,11 +72,11 @@ Since it supports the WASI specification, C can be used on the Fermyon Platform
...
+At the time of this writing, the latest version is `20.0`:
 
 ```console
-$ curl -O -sSL https://github.com/WebAssembly/wasi-sdk/releases/download/wasi-sdk-14/wasi-sdk-14.0-macos.tar.gz
-$ tar xf wasi-sdk-14.0-macos.tar.gz
+$ curl -O -sSL https://github.com/WebAssembly/wasi-sdk/releases/download/wasi-sdk-20/wasi-sdk-20.0-linux.tar.gz
+$ tar -xzf wasi-sdk-20.0-linux.tar.gz
 ```
 
 Now we can write a simple C program using WASI. Here is the text ouf `hello.c`:
```
Comment (tpmccallum) [inline content/wasm-languages/c-lang.md:82]:
```suggestion
Now we can write a simple C program using WASI. Here is the text our `hello.c`:
```

**Review by tpmccallum — COMMENTED** [review]
Typo

File: content/wasm-languages/c-lang.md [hunk 2]
```diff
@@ -72,14 +72,14 @@ Since it supports the WASI specification, C can be used on the Fermyon Platform
...
 
 ```console
-$ curl -O -sSL https://github.com/WebAssembly/wasi-sdk/releases/download/wasi-sdk-14/wasi-sdk-14.0-macos.tar.gz
-$ tar xf wasi-sdk-14.0-macos.tar.gz
+$ curl -O -sSL https://github.com/WebAssembly/wasi-sdk/releases/download/wasi-sdk-20/wasi-sdk-20.0-linux.tar.gz
+$ tar -xzf wasi-sdk-20.0-linux.tar.gz
 ```
 
-Now we can write a simple C program using WASI. Here is the text ouf `hello.c`:
+Now we can write a simple C program using WASI. Here is the text our `hello.c`:
```
Comment (tpmccallum) [inline content/wasm-languages/c-lang.md:82]:
```suggestion
Now we can write a simple C program using WASI. Here is the text of our `hello.c` file:
```

**Review by itowlson — APPROVED** [review]

**Comment by tpmccallum** [comment]
Thanks @itowlson

**Pull request merged**